### ISSUE TITLE
Porntrex account subscriptions

### ISSUE THREAD

**Issue by jp194978** [dobbelina/repository.dobbelina#735]
Porntrex account( subscription) Manage subscriptions not working... #

**Comment by DJMix-I-Can** [comment]
I just tested it on Kodi 19.4 & it's working. If you're still on Leia 18.9, consider on updating. Cumination was created for Kodi 19 Matrix, I need to rephrase on updating Kodi. Kodi Leia 18.9 is using Python 2. You cannot download Kodi 19.X and upgrade your current Kodi 18.9 installation. It has to be a fresh install. Yes, you're going to have to reconfigure everything from scratch. It's a Python incompatibly issue.

**Comment by jp194978** [comment]
i¨m on Kodi 19.3 will udpate to 19.4  thanks

**Comment by Gujal00** [comment]
Cumination is meant to work on both Kodi 18 and 19 and all the code has been done using the six library to make the code run on both Python 2 and 3. If there are issues happening in Leia but not in Matrix, please report as it may be string handling issue in code and in most cases fixable

**Comment by Gujal00** [comment]
@jp194978 See if porntrex is accessible from your browser without any cloudflare message showing up before opening the site
If it works without issues in browser, then do the following

1.     Install Kodi Logfile Uploader addon from Offcial Kodi Repository under Program add-ons.
2.     Enable Debug logging, (Settings - System - Logging - Enable Debug Logging)
3.     Restart Kodi (Important)
4.     Go into Cumination and open the site (wait for the error message)
5.     Come out of cumination
6.     Disable Debug logging, (Settings - System - Logging - Enable Debug Logging off)
7.     launch Kodi Logfile uploader and upload your log. This will give you a url
8.     provide the url from step 7 here

@dobbelina can you add this info on the issue template, people are still opening issues without attaching a debug log

**Comment by jp194978** [comment]
kodi logfile for porntrex manage subscription error : https://paste.kodi.tv/upoyurodiq  
thanks

**Comment by jp194978** [comment]
> @jp194978 See if porntrex is accessible from your browser without any cloudflare message showing up before opening the site If it works without issues in browser, then do the following
> 
>     1. Install Kodi Logfile Uploader addon from Offcial Kodi Repository under Program add-ons.
> 
>     2. Enable Debug logging, (Settings - System - Logging - Enable Debug Logging)
> 
>     3. Restart Kodi (Important)
> 
>     4. Go into Cumination and open the site (wait for the error message)
> 
>     5. Come out of cumination
> 
>     6. Disable Debug logging, (Settings - System - Logging - Enable Debug Logging off)
> 
>     7. launch Kodi Logfile uploader and upload your log. This will give you a url
> 
>     8. provide the url from step 7 here
> 
> 
> @dobbelina can you add this info on the issue template, people are still opening issues without attaching a debug log

Kodi log file for porntrex manage subscritions error...https://paste.kodi.tv/kifijobafe.kodi.....

**Comment by Gujal00** [comment]
@erosvece
Any insight on this? I do not have porntrex account to test
```
2022-03-31 19:06:41.295 T:12132   ERROR <general>: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
       - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
       Error Type: <class 'IndexError'>
       Error Contents: list index out of range
       Traceback (most recent call last):
       File "C:\Users\Jorge\AppData\Roaming\Kodi\addons\plugin.video.cumination\default.py", line 155, in <module>
       sys.exit(main())
       File "C:\Users\xxxxx\AppData\Roaming\Kodi\addons\plugin.video.cumination\default.py", line 148, in main
       url_dispatcher.dispatch(mode, queries)
       File "C:\Users\xxxxx\AppData\Roaming\Kodi\addons\plugin.video.cumination\resources\lib\url_dispatcher.py", line 119, in dispatch
       cls.func_registry[mode](*args, **kwargs)
       File "C:\Users\Jorge\AppData\Roaming\Kodi\addons\plugin.video.cumination\resources\lib\sites\porntrex.py", line 333, in PTSubscriptions
       id = img.split('/')[5]
      IndexError: list index out of range
-->End of Python script error report<--
```

**Comment by dobbelina** [comment]
A fix will be in the next 1.1.39 release https://github.com/dobbelina/repository.dobbelina/commit/bb139ca2e1f1e55b71181a895b294fa7bd9b9fa6

**Comment by jp194978** [comment]
Thanks for the fix

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
